.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +18,13 @@
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
+- [ ] I have determined whether the [framework documentation](https://docs.signac.io/) requires an update (see guidance below). If so, I have created a separate pull request to update it on [signac-docs](https://github.com/glotzerlab/signac-docs) at this link: glotzerlab/signac-docs/pull#**XXX**
+<!-- The framework documentation must be updated if:
+        This PR will change user-facing features (labeled "enhancement").
```

Comment:
That wasn't obvious to me. Could we rename the label "enhancement" to "internal"?

Reply:
I was thinking that any PR not marked as a bug fix or docs update would be an enhancement. (Even bug fixes are a kind of enhancement)

Reply:
> I was thinking that any PR not marked as a bug fix or docs update would be an enhancement.

That's my stance as well. I just wanted to change the language "will change user-facing features" because it's not broad enough.